preferencesfx/src/main/java/com/dlsc/preferencesfx/view/PreferencesFxDialog.java

```diff
@@ -43,9 +44,10 @@
    * @param model             the model of PreferencesFX
    * @param preferencesFxView the master view to be display in this {@link DialogPane}
```

Comment:
javadoc for modal is missing

Reply:
I have added the missing part of the javadoc.